kaggle_environments/envs/mab/mab.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "mab",
+  "title": "Adversarial Multi-armed Bandit",
+  "description": "Adversarial Multi-armed Bandit",
+  "version": "1.0.0",
+  "agents": [2, 3, 4, 5, 6, 7, 8],
+  "configuration": {
+    "episodeSteps": 1200,
+    "actTimeout": 0.25,
+    "banditCount": {
+      "description": "Number of bandits available to choose from. Max action is this number -1.",
+      "type": "integer",
+      "minimum": 2,
+      "default": 12
+    },
+    "decayRate": {
+      "description": "Rate that reward chance threshold increases per step that a bandit is chosen by an agent.",
+      "type": "number",
+      "minimum": 0,
+      "default": 0.03
+    },
+    "sampleResolution": {
+      "description": "Maximum value that can be returned by a bandit.",
+      "type": "integer",
+      "minimum": 1,
+      "default": 100
+    }
+  },
+  "reward": {
+    "description": "The total reward each player has accumulated from the bandits across all steps.",
+    "type": "number",
+    "minimum": 0,
+    "default": 0
+  },
+  "observation": {
+    "remainingOverageTime": 60,
+    "reward": {
+      "description": "Current reward of the agent.",
+      "type": "number",
+      "minimum": 0,
+      "default": 0
+    },
+    "lastActions": {
+      "description": "Bandits chosen by each agent on the last step. None on the first step.",
```

Comment:
Sometimes they're called bandits and sometimes thresholds, but both references will be replaced when we add branding.